Gemfile

```diff
@@ -6,7 +6,7 @@ ruby '~> 2.7.3'
 gem 'rails', '~> 6.1.3'
 
 # Variables can be overridden for local dev in Gemfile-dev
-@doc_auth_gem ||= { github: '18F/identity-doc-auth', tag: 'v0.9.3' }
+@doc_auth_gem ||= { github: '18F/identity-doc-auth', tag: 'v0.10.0' }
```

Comment:
Corresponds to https://github.com/18F/identity-doc-auth/pull/23, which updates that gem to send back DOB/expiration in international format